includes/class-rest.php

```
@@ -840,6 +876,194 @@ public function run_performance_scan( \WP_REST_Request $request ): \WP_REST_Resp
+		 * @since 1.7.0
+		 * @return \WP_REST_Response The response object.
+		 */
+		public function get_telemetry( \WP_REST_Request $request ): \WP_REST_Response {
+			require_once WPPO_PLUGIN_PATH . 'includes/class-telemetry-table.php';
+			$params = $request->get_params();
+			$url    = isset( $params['url'] ) ? esc_url_raw( $params['url'] ) : '';
+
+			$rows = Telemetry_Table::get_rows( $url );
+
+			return $this->send_response( $rows );
+		}
```

PHPMD (2.15.0)

[error] The method get_telemetry is not named in camelCase. (undefined)

(CamelCaseMethodName)